src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts

```diff
@@ -2601,7 +2602,8 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
 			width: this._dimension?.width ?? 0,
 			height: this._dimension?.height ?? 0,
 			scrollHeight: this._list?.getScrollHeight() ?? 0,
-			fontInfo: this._fontInfo!
+			fontInfo: this._fontInfo!,
+			stickyHeight: this._notebookStickyScroll.getCurrentStickyHeight()
```

Comment:
since we define `_notebookStickyScroll` as `_notebookStickyScroll!`, ts won't warn us that it can be undefined, but it actually can. so we want to use `this._notebookStickyScroll?`